.gitignore

```diff
@@ -35,3 +35,98 @@ Carthage/Build
 .idea/
 *.iml
 
+# Created by https://www.gitignore.io/api/swift,xcode,swiftpm
```

Comment:
there must be duplication above, right?

Reply:
Yep, good catch, I cleanup up the whole `.gitignore` [here](https://github.com/thoughtbot/Swish/pull/124/commits/e768499a1500e5e7d734906b2b0fded15bfea3c0)